### PR TITLE
Update SISTEM KPR

### PR DESCRIPTION
**Pull request by AW0x**
Update: Sorting data

### CODE REVIEW

**Review by mortekianond — APPROVED** [review]

**Review by mortekianond — APPROVED** [review]

**Review by AW0x — COMMENTED** [review]
Update: Sorting data

**Review by AW0x — COMMENTED** [review]
Update

File: SISTEM KPR [hunk 1]
```diff
@@ -404,14 +397,18 @@ void sort_harga_descending(){
...
 			}
 		}
+		if(position != i){
+			strcpy(tukar, rumah[i].type);
+			strcpy(rumah[i].type, rumah[position].type);
+			strcpy(rumah[position].type, tukar);
+		}
 	}
 	
 	// Harga
```
Comment (mhmdrandy) [inline SISTEM KPR:414]: /

**Review by mhmdrandy — COMMENTED** [review]
.,

**Pull request merged**